modules/Bio/EnsEMBL/Variation/DBSQL/CADDAnnotationAdaptor.pm

```diff
@@ -199,10 +197,9 @@ sub fetch_by_id {
 
 sub fetch_all {
```

Comment:
Same as above: not sure we need to write this as this method is defined in the `BaseAnnotationAdaptor`

Reply:
Same here, I want to keep the documentation of the method in this module.